microshift_running_apps/microshift-operators.adoc

```diff
@@ -10,9 +10,67 @@ You can use Operators with {microshift-short} to create applications that monito
 
-{microshift-short} applications are generally expected to be deployed in static environments. However, Operators are available if helpful in your use case. To determine an Operator's compatibility with {microshift-short}, check the Operator's documentation.
+{microshift-short} applications are generally expected to be deployed in static environments. However, Operators are available if helpful in your use case. To determine the compatibility of an Operator with {microshift-short}, check the Operator documentation.
 
-[id="how-to-install-operators_{context}"]
+[id="microshift-operators-installation-paths_{context}"]
 == How to install Operators in {microshift-short}
 
-To minimize the footprint of {microshift-short}, Operators are installed directly with manifests instead of using the Operator Lifecycle Manager (OLM). You can use the `kustomize` configuration management tool with {microshift-short} to deploy an application. Use the same steps to install Operators with manifests. Read xref:../microshift_running_apps/microshift-applications.adoc#microshift-manifests-overview_applications-microshift[Using Kustomize manifests to deploy applications] for more information about manifests.
\ No newline at end of file
+There are two ways to install Operators for your {microshift-short} clusters:
+
+* Operators can be installed directly with manifests. You can use the `kustomize` configuration management tool with {microshift-short} to deploy an application. Use the same steps to install Operators with manifests. See xref:../microshift_running_apps/microshift-applications.adoc#microshift-manifests-overview_applications-microshift[Using Kustomize manifests to deploy applications] for more information about manifests.
+
+* You can also install OLM-based Operators to a {microshift-short} cluster using Using Operator Lifecycle Manager (OLM).
+//TODO: see OSDOCS-8989 for summary and xref after PR is merged
+
+[id="microshift-olm"]
+== Using Operator Lifecyle Manager
+OLM is used in {microshift-short} for installing and running optional link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.14/html/architecture/control-plane#olm-operators_control-plane[add-on Operators].
+
+* Cluster Operators as applied in {ocp} are not used in {microshift-short}.
+* You must use the CLI to conduct OLM activities with {microshift-short}. The console and OperatorHub GUIs are not available, but you can take all the same actions with the CLI.
+
+[IMPORTANT]
+====
+Before using an Operator, verify with the provider whether the Operator is supported on {product-title}.
+====
+
+[id="microshift-file-based-olm-catalogs"]
+== File-based Operator catalogs
+
+To run {microshift-short} with your own curated catalogs and OLM, use the file-based catalog structure. For a complete explanation of the file-based structure, see the "Additional resources" section that immediately follows this one.
```

Comment:
We might want to explicitly call out how to create custom catalogs from the default ocp shipped catalogs? In most cases, I would expect customers aren't doing this from scratch but instead are probably using the already shipped manifests that operators use.

Today, we have documentation for that via oc-mirror, because making custom catalogs based on the default catalogs is needed for disconnected, but you can in theory do the same thing regardless of whether you're actually mirroring to a cluster that doesn't have external network access.